python/services/partitionerservice/dmod/partitionerservice/service.py

```diff
@@ -19,7 +19,7 @@
 from uuid import uuid4
 
 logging.basicConfig(
-    level=logging.DEBUG,
+    level=logging.INFO,
```

Comment:
This might need to help boost the priority of moving some logging logic to `dmod.core` or elsewhere so this call doesn't have to be repeated elsewhere. I swear I've seen this exact wording elsewhere. If this one is changed to `INFO`, but another isn't, the level will still be `DEBUG` if that one is hit first (I believe, anyways). If `dmod.partitionerservice.__main__` calls a file that defines `logging.basicConfig`_before_ importing `service`, this doesn't amount to much.

Reply:
I think we have to kick that can a little further down the road, but you are correct in that it needs to be reworked.